reana_db/models.py

```diff
@@ -416,6 +416,7 @@ class Workflow(Base, Timestamp, QuotaBase):
     reana_specification = Column(JSONType)
     input_parameters = Column(JSONType)
     operational_options = Column(JSONType)
+    complexity = Column(String, default="[]")
```

Comment:
Any particular reason why we store complexity as `String`? We could use SQLAlchemy [`ARRAY`](https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.ARRAY), or even JSON.

Reply:
I was a bit hesitating to try it since `ARRAY` is only supported by `PostgreSQL`. However just tried it worked as expected